### PR TITLE
Form: add e2e tests handling question creation

### PR DESCRIPTION
**Pull request by AdrienClairembault**
Test the creation of questions from the form editor user interface.

I've added some common boilerplate code related to form creation into new cypress commands (will retrofit existing tests later).
These commands have their own file to avoid polluting the general `command.js` file.

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |

### CODE REVIEW

File: tests/cypress/support/commands/form.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+Cypress.Commands.add('createFormWithAPI', (
```
Comment (cconard96) [inline tests/cypress/support/commands/form.js:34]: Please add JSDoc comments for IDE recognition and documentation.

**Review by cconard96 — CHANGES_REQUESTED** [review]

File: tests/cypress/support/commands/form.js (hunk 1 above)
Comment (AdrienClairembault) [reply tests/cypress/support/commands/form.js:34]: Unless I am missing something, VScode does not support JSDoc for intellisense on custom commands.
The documentation seems to indicate that the correct way to implement IDE recognition is to add typescript definition file for each command file.

I also noticed that we are lacking the basic `jsconfig.json` file to enable cy intellisense (which I can add it in another PR).

Comment (cconard96) [reply tests/cypress/support/commands/form.js:34]:
```js
/**
 * @memberof Cypress.Chainable.prototype
 * @method createFormWithAPI
 * @description Create a form using the GLPI API
 */
```

Adding comments like this is sufficient. If VSCode doesn't read JSDoc, that seems like a big missing feature. In PHPStorm it works OK.

What is the `jsconfig.json` file supposed to do? I don't notice any missing IDE autocomplete or documentation.

Comment (cconard96) [reply tests/cypress/support/commands/form.js:34]: Oh, I see. Only some IDEs like VSCode require the extra file to discover symbols...
https://docs.cypress.io/guides/tooling/IDE-integration#Reference-type-declarations-via-jsconfig

Comment (cconard96) [reply tests/cypress/support/commands/form.js:34]: Triple-slash is TypeScript only, no? It's a compiler-time directive.
The new commit does add autocomplete info, but that TypeScript file is doing all the work on its own. PHPStorm is smart enough to find and match `*.d.ts` files on its own.

This was discussed before (either when Vue was added or Cypress, I don't remember) that TS would add too much complexity and there aren't enough people even comfortable with vanilla JS currently.

**Review by cconard96 — APPROVED** [review]

**Review by orthagh — APPROVED** [review]

**Pull request merged**